lib/src/kbasesearchengine/main/NarrativeInfoDecorator.java

```diff
@@ -81,12 +85,18 @@ public SearchTypesOutput searchTypes(final SearchTypesInput params, final String
     public SearchObjectsOutput searchObjects(final SearchObjectsInput params, final String user)
             throws Exception {
         SearchObjectsOutput searchObjsOutput = searchInterface.searchObjects(params, user);
+        if (searchObjsOutput.getAdditionalProperties().get(removedGuids) == null &&
+                System.getProperty(removedGuidsEnv) != null &&
```

Comment:
You can just do `'true'.equals(System.getProperty(removedGuidsEnv)` to avoid the null check